.connector-store/meta.json

```diff
@@ -14,6 +14,13 @@
         "IS 7.0.0",
         "IS 7.1.0"
       ]
+    },
+    {
+      "tagName": "v1.0.5",
```

Comment:
This has not been released yet tight?

Reply:
addressed with https://github.com/wso2-extensions/identity-outbound-auth-iproov/pull/14/commits/220a195e716f18409763ced0efb143eac8544347